.circleci/config.yml

```diff
@@ -8,16 +8,6 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker
-      - restore_cache:
```

Comment:
So this broke stuff huh.

Reply:
Yes. They changed it recently; requires an opt-in; the condo image itself is cached; but dependencies aren’t... since condo doesn’t have any dependencies; it doesn’t matter.